nasa/harmony-regression-tests
# DAS-1935 - Enable RSSMIF16D production HOSS regression tests.

## Description

This PR updates the HOSS notebook so that the RSSMIF16D regression tests can be run against production. This collection is now available in production, and GHRC has associated it with HOSS. However, the GES DISC collections in the notebook still aren't configured in HOSS, so I've made the decision to run tests a little less global.

## Jira Issue ID

DAS-1935 (not an A/C, but discovered while testing)

## Local Test Steps

* Pull this branch.
* Run the notebook against UAT and Production, by setting the appropriate `HARMONY_HOST_URL`.
* Note that the requests that use `hoss_info['collection']` run and pass, but that other requests are skipped.
* **Note - I've seen some slightly slow Harmony performance in production, where requests have been queued for a while before executing, but the requests were passing when they eventually ran.**

## PR Acceptance Checklist
* ~~Acceptance criteria met~~
* ~~Documentation updated (if needed)~~
